src/model/document/elements/expression-element.ts

```diff
@@ -250,8 +255,32 @@ export class ExpressionElement extends QuantumElement {
 export const ExpressionElementType: QuantumElementType<ExpressionElement, typeof ExpressionElement, typeof ElementType> = {
   typeName: ElementType,
   elementType: ExpressionElement,
-  serializeElement: (element) => null,
-  deserializeElement: (stuff) => null as any,
+  serializeElement: (element: ExpressionElement) => {
+    const serializedElement = {
+      id: element.id, // unnecessary?
+      position: { x: element.position.value.x, y: element.position.value.y },
+      size: { x: element.size.value.x, y: element.size.value.y },
+      resizable: element.resizable.value,
+      // expression element properties
+      expression: element.expression.value,
+    }
+    return serializedElement
+  },
+  deserializeElement: (element) => {
+    // null as any
+    const expressionElement = {
+      // document-element properties
+      // id:
+      creationOptions: {
+        position: new Vector2(element?.position?.x, element?.position?.y),
+        size: new Vector2(element?.size.x, element?.size.y),
+        resizable: element?.resizable,
+      },
+      // expression element properties
+      expression: element.expression,
+    }
+    return expressionElement
+  },
```

Comment:
Sweet, that looks just like how I'd have imagined it! I think we can get away without serializing the IDs for now.

Though, actually, leave them in there, doesn't hurt to have them for later. If we ever need to reference elements from other elements, we'll probably need the IDs. (e.g. referencing an expression element from a graph element and saying 'please plot this function')

Reply:
Yea, that's what I figured. I guess I can see about creating an element with a specified id.